tests/phpunit/unit/class-test-case.php

```diff
@@ -37,6 +37,11 @@ protected function setUp() {
 
 			return $path;
 		} );
```

Comment:
Ignore this function.  It's part of PR #92.  It's shown here as I built this PR off of PR #92's branch.